test/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth.test.cpp

```diff
@@ -0,0 +1,135 @@
+
+#include <OpenSpaceToolkit/Core/FileSystem/File.hpp>
+#include <OpenSpaceToolkit/Core/FileSystem/Path.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Array.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Tuple.hpp>
+
+#include <Global.test.hpp>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+TEST (OpenSpaceToolkit_Physics_Environment_Atmospheric_Earth, Constructor)
+{
+
+    using ostk::core::fs::Path ;
+    using ostk::core::fs::Directory ;
+
+    using EarthAtmosphericModel = ostk::physics::environment::atmospheric::Earth ;
+
+    {
+
+        EXPECT_NO_THROW(EarthAtmosphericModel earthAtmosphericModel(EarthAtmosphericModel::Type::Exponential)) ;
+
+    }
+
+    {
+
+        EXPECT_NO_THROW(EarthAtmosphericModel earthAtmosphericModel(EarthAtmosphericModel::Type::Exponential, Directory::Path(Path::Parse("/app/test/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth")))) ;
+
+    }
+
+    {
+
+        EXPECT_NO_THROW(EarthAtmosphericModel earthAtmosphericModel(EarthAtmosphericModel::Type::Exponential, Directory::Path(Path::Parse("/does/not/exist")))) ;
+
+    }
+
+}
+
+
+TEST (OpenSpaceToolkit_Physics_Environment_Atmospheric_Earth, GetType)
+{
+
+    using EarthAtmosphericModel = ostk::physics::environment::atmospheric::Earth ;
+
+    {
+
+        EXPECT_EQ(EarthAtmosphericModel::Type::Exponential, EarthAtmosphericModel(EarthAtmosphericModel::Type::Exponential).getType()) ;
+
+    }
+
+}
+
+
+TEST (OpenSpaceToolkit_Physics_Environment_Atmospheric_Earth, GetDensityAt)
+{
+
+    using ostk::core::types::Real ;
+    using ostk::core::types::String ;
+    using ostk::core::ctnr::Tuple ;
+    using ostk::core::ctnr::Array ;
+    using ostk::core::fs::Path ;
+    using ostk::core::fs::Directory ;
+
+    using ostk::physics::coord::spherical::LLA ;
+    using ostk::physics::units::Length ;
+    using ostk::physics::units::Angle ;
+    using ostk::physics::time::Instant ;
+    using EarthAtmosphericModel = ostk::physics::environment::atmospheric::Earth ;
+
+    {
+
+        static const Array<Tuple<EarthAtmosphericModel::Type, LLA, Instant, Real, Real>> testCases =
+        {
+
+            { EarthAtmosphericModel::Type::Exponential, LLA( Angle::Degrees(35.076832), Angle::Degrees(-92.546296), Length::Kilometers(123.0) ), Instant::J2000(), 1.77622e-08, 1e-13 },
+            { EarthAtmosphericModel::Type::Exponential, LLA( Angle::Degrees(35.076832), Angle::Degrees(-92.546296), Length::Kilometers(499.0) ), Instant::J2000(), 7.08245e-13, 1e-15 },
+            { EarthAtmosphericModel::Type::Exponential, LLA( Angle::Degrees(35.076832), Angle::Degrees(-92.546296), Length::Kilometers(501.0) ), Instant::J2000(), 6.85869e-13, 1e-15 },
+            { EarthAtmosphericModel::Type::Exponential, LLA( Angle::Degrees(35.076832), Angle::Degrees(-92.546296), Length::Kilometers(550.0) ), Instant::J2000(), 3.18278e-13, 1e-15 }
```

Comment:
Unfortunately, GMAT doesn't define unit tests for this part. I suspect there are some there are some example values in Vallado, but since the model is simple enough, I just hand calculated a few scenarios. 
Agree though about a comment here so they're not just magic numbers.

Reply:
Awesome sounds good.